msmbuilder/featurizer/multichain.py

```diff
@@ -0,0 +1,510 @@
+        chain in the trajectory containing the protein of interest. 'auto'
+        chooses the longest alpha-carbon containing chain. for chains with
+        an identical number of atoms, 'auto' will choose the first chain
+        in the topology file.
+    ligand_chain : int or 'auto', default='auto'
+        chain in the trajectory containing the ligand of interest. 'auto'
+        chooses the chain containing the most atoms, not to exceed 100
+        atoms, that is not already the protein chain. for chains with
+        an identical number of atoms, 'auto' will choose the first chain
+        in the topology file.
+    reference_frame : md.Trajectory, default=None
+        single-frame conformation to get chain information; also defines
+        the binding pocket if specified
+    contacts : np.ndarray or 'all'
+        array containing (0-indexed) indices of the residues to compute the
+        contacts for. (e.g. np.array([[0, 10], [0, 11]]) would compute
+        the contact between residue 0 and residue 10 as well as
+        the contact between residue 0 and residue 11.) [NOTE: if no
+        array is passed then 'all' contacts are calculated. This means
+        that the result will contain contacts between all protein
+        residues and the ligand.]
+    scheme : {'ca', 'closest', 'closest-heavy'}, default='closest-heavy'
+        scheme to determine the distance between two residues:
+            'ca' : distance between two residues is given by the distance
+                between their alpha carbons. Only allowed if the ligand
+                contains an alpha carbon.
+            'closest' : distance is the closest distance between any
+                two atoms in the residues
+            'closest-heavy' : distance is the closest distance between
+                any two non-hydrogen atoms in the residues
+    binding_pocket : float or 'all', default='all'
+        nanometer cutoff to define a binding pocket; if defined, only
+        protein atoms within the threshold distance according to the
+        topology file will be included
+
+    """
+
+    def __init__(self, protein_chain='auto', ligand_chain='auto',
+                 reference_frame=None, contacts='all', 
+                 scheme='closest-heavy', binding_pocket='all'):
+        super(LigandContactFeaturizer, self).__init__(
+                    protein_chain=protein_chain, ligand_chain=ligand_chain,
+                    reference_frame=reference_frame)
+        self.contacts = contacts
+        self.scheme = scheme
+        self.binding_pocket = binding_pocket
+
+        self.contacts = self._get_contact_pairs(reference_frame, contacts)
+
+    def _get_contact_pairs(self, traj, contacts):
+        if self.scheme=='ca':
+            if not any(a for a in traj.top.chain(ligand_chain).atoms
+                       if a.name.lower() == 'ca'):
+                raise ValueError("Bad scheme: the ligand has no alpha carbons")
```

Comment:
One of the ligands I'm working with is glutamine, which does. Seems better to allow for the rare case than to get rid of the option.